web/src/test/java/org/springframework/security/web/savedrequest/CookieRequestCacheTests.java

```diff
@@ -153,6 +171,7 @@ private MockHttpServletRequest requestToSave() {
 		request.setScheme("https");
 		request.setServerName("abc.com");
 		request.setRequestURI("/destination");
+		request.setServletPath("/destination");
```

Comment:
We should not have to set the servlet path for this test to pass.

Reply:
I fixed these issues, please check.